src/utils/mod.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2017 fd developers
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0>
+// or the MIT license <LICENSE-MIT or http://opensource.org/licenses/MIT>,
+// at your option. All files in the project carrying such
+// notice may not be copied, modified, or distributed except
+// according to those terms.
+
+use std::path::Path;
+use std::iter::Iterator;
+use std::ascii::AsciiExt;
```

Comment:
This import seems to be unused.

Reply:
Unfortunately, this is needed for Rust `1.1.20` (see [this build](https://travis-ci.org/sharkdp/fd/jobs/328591730)) because `u8` and `u16` are missing the `eq_ignore_ascii_case` method.

And when it rains, it pours: there is no `cfg` attribute depending on Rust version.

Reply:
Do you know when this was changed? I'd be okay with bumping the minimum-required Rust version.

Reply:
The import is unneeded since `1.1.23`.

Reply:
Thanks for tracking this down. In this case, could we please change this to
``` rust